bundles/org.openhab.binding.plex/src/main/resources/OH-INF/i18n/plex.properties

```diff
@@ -0,0 +1,38 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
+# FIXME: please do not add the file to the repo if you add or change no content
```

Comment:
Can these comments be removed?

You could create this default translation file by running `mvn org.openhab.core.tools:i18n-maven-plugin:3.3.0:generate-default-translations` in your binding's directory. Then, [Crowdin](https://crowdin.com/project/openhab-addons) will pick up the English "translation" and users can help to translate your binding in Crowdin. If you want you can use Crowdin, too to translate your binding as soon as this PR is merged.